site/public/js/grade-inquiry.js

```diff
@@ -1,6 +1,27 @@
 /* global buildCourseUrl, WebSocketClient */
 /* exported initGradingInquirySocketClient, onComponentTabClicked, onGradeInquirySubmitClicked, onReady, onReplyTextAreaKeyUp */
 
+function saveDraft() { 
```

Comment:
```suggestion
function saveDraft() {
```
Remove the trailing space here.